module/VuFind/src/VuFind/Controller/AbstractRecord.php

```diff
@@ -258,12 +251,6 @@ public function deletetagAction()
      */
     public function homeAction()
     {
-        // Save statistics:
-        if ($this->logStatistics) {
-            $this->getServiceLocator()->get('VuFind\RecordStats')
```

Comment:
I think the VuFind\SearchStats and VuFind\RecordStats definitions probably need to be removed from module.config.php. I don't see any diffs taking them out yet.